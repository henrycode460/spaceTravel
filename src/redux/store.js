import { configureStore } from '@reduxjs/toolkit';
import { combineReducers } from 'redux';
import missionsReducer from './missions/missions';
import rockertsReducer from './rockets/rockets';

const rootStore = combineReducers({
   missions: missionsReducer,
   rockets: rockertsReducer,
});

const store = configureStore({
  reducer: rootStore,
});

export default store;
