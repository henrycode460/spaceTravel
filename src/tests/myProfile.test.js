import TestRenderer from 'react-test-renderer';
import { Provider } from 'react-redux';
import store from '../redux/store.js';
import MyProfile from "../components/myProfile";

it('Testing my profile component:', () => {
  const tree = TestRenderer.create(
    <Provider store={store}>
      <MyProfile />
    </Provider>).toJSON();
  expect(tree).toMatchSnapshot();
});