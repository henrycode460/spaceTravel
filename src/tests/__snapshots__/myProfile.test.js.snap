// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Testing my profile component: 1`] = `
<div
  className="myProfile"
>
  <div
    className="container"
  >
    <h2>
      My Missions
    </h2>
    <ul />
  </div>
  <div
    className="container"
  >
    <h2>
      My Rockets
    </h2>
    <ul />
  </div>
</div>
`;
